src/System.Windows.Forms/src/System/Windows/Forms/ToolStripSplitButton .ToolStripSplitButtonUiaProvider.cs

```diff
@@ -52,9 +55,16 @@ internal override UiaCore.ExpandCollapseState ExpandCollapseState
                 }
             }
 
-            internal override UiaCore.IRawElementProviderFragment? FragmentNavigate(UiaCore.NavigateDirection direction)
+            internal override UiaCore.IRawElementProviderFragment? FragmentNavigate(Interop.UiaCore.NavigateDirection direction)
```

Comment:
Looks like it's not necessary
```suggestion
            internal override UiaCore.IRawElementProviderFragment? FragmentNavigate(UiaCore.NavigateDirection direction)
```